src/components/Company/blocks/Details.tsx

```diff
@@ -0,0 +1,61 @@
+import { useMemo } from "react";
+import { Stack } from "@deskpro/deskpro-ui";
+import { Title, Property, Member, LinkIcon } from "@deskpro/app-sdk";
+import { getAddress, getExternalLinks } from "@/utils";
+import { CopperLogo, DPNormalize } from "@/components/common";
+import type { FC } from "react";
+import type { Maybe } from "@/types";
+import type { Company, Account, Contact } from "@/services/copper/types";
+
+export type Props = {
+  company: Company;
+  account: Maybe<Account>;
+  primaryContact: Maybe<Contact>;
+};
+
+const Details: FC<Props> = ({ company, account, primaryContact }) => {
+  const link = useMemo(() => {
+    return getExternalLinks.company(account?.id, company.id);
+  }, [account?.id, company.id]);
+  const contactLink = useMemo(() => {
+    return getExternalLinks.contact(account?.id, primaryContact?.id);
+  }, [account?.id, primaryContact?.id]);
+  const phones = useMemo(() => {
+    return company.phone_numbers?.map(({ number }) => number).join(`,\n`);
+  }, [company.phone_numbers]);
+  const sites = useMemo(() => {
+    return company.websites?.map(({ url }) => url).join(",\n");
+  }, [company.websites]);
```

Comment:
No need for memos here, operations are simple.

Reply:
done